src/js/msp/MSPHelper.js

```diff
@@ -764,25 +764,32 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 RX_CONFIG.spektrum_sat_bind = data.readU8();
                 RX_CONFIG.rx_min_usec = data.readU16();
                 RX_CONFIG.rx_max_usec = data.readU16();
+                RX_CONFIG.rcInterpolation = 0;
+                RX_CONFIG.rcInterpolationInterval = 0;
+                RX_CONFIG.airModeActivateThreshold = 0;
                 if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
                     RX_CONFIG.rcInterpolation = data.readU8();
                     RX_CONFIG.rcInterpolationInterval = data.readU8();
                     RX_CONFIG.airModeActivateThreshold = data.readU16();
+                    RX_CONFIG.rxSpiProtocol = 0;
+                    RX_CONFIG.rxSpiId = 0;
+                    RX_CONFIG.rxSpiRfChannelCount = 0;
+                    RX_CONFIG.fpvCamAngleDegrees = 0;
```

Comment:
You don't need to initialize this, in theory all are zero in the definition of the object. Did you observe some strange behavior that needed that?

Reply:
@McGiverGim I did this in an attempt to cleanup the code and remove the else statements.  If you don't like that I can switch it back but, personally I think it looks a lot cleaner to do this and it shouldnt have any negative effects.  Just let me know though if you prefer having the else's in there I'll drop that change

Reply:
What I say is that you don't need the `else` neither. You must assign value only to the fields when the MSP protocol send it. If you don't assign them, they are zero by default (you are doing it in the `fc.js` file).

Reply:
Even better! I'll remove it all together and test it out